debs/SPECS/3.9.4/wazuh-agent/debian/changelog

```diff
@@ -0,0 +1,241 @@
+wazuh-agent (3.9.3-RELEASE) stable; urgency=low
```

Comment:
The changelog lines form Wazuh v3.9.4 are missing.

Reply:
https://github.com/wazuh/wazuh-packages/pull/228/commits/ca02cfa00eeb057c2e184ebf3ccea1922109ac1a